### ISSUE TITLE
This literally does not answer what I am asking.

### ISSUE THREAD

**Issue by LiterallyMadeThisAccountForOneQuestion** [Realm667/WolfenDoom#1281]
That happens because:
1. GZDoom does not support [clamping](https://learnopengl.com/img/getting-started/texture_wrapping.png) individual textures, unless a shader is used to force the effect, and that isn't the case currently.
2. The model uses [this texture](https://github.com/Realm667/WolfenDoom/blob/master/models/lightbeam.png), which tiles seamlessly horizontally, but not vertically.

_Originally posted by @Talon1024 in https://github.com/Realm667/WolfenDoom/issues/1280#issuecomment-1347701820_

**Comment by LiterallyMadeThisAccountForOneQuestion** [comment]
I am not trying to be rude. I thank you guys for making this game. But I want to know if there is a way for this to be fixed. Your statement contains no certain laymans terms on how to fix this.

**Comment by Tormentor667** [comment]
That’s because we are not responsible for the engine. If you want this to be fixed you need to ask for it over at zdoom.org :)
